caluma/caluma_form/schema.py

```diff
@@ -82,6 +87,25 @@ class QuestionJexl(graphene.String):
 )
 
 
+WorkItemButtonActionType = graphene.Enum(
+    "WorkItemButtonActionType",
+    [(key.upper(), key) for key, desc in models.Question.ACTION_TYPE_CHOICES_TUPLE],
+)
+WorkItemButtonColor = graphene.Enum(
```

Comment:
Can we rename this to just `ButtonColor`? Even if it's currently only used on workitem buttons, it may be useful in other situations as well, so no need to have a over-specific name